doc/_src_docs/applications/ego.rstx

```diff
@@ -159,6 +159,10 @@ Beside the Expected Improvement, the implementation here offers two other infill
 Regarding the parallel execution, one can implement specific multiprocessing by deriving the _Evaluator_ interface
 and overriding the default implementation of the _run(fun, x)_ method. The default implementation simply runs _fun(x)_.
 
+Regardless the others parameters, you can specify a mixed surrogate model to make mixed optimization. [8]_
+
+You can penalize already evaluated point in EI via tunneling (not recommanded for high dimensional problems).
```

Comment:
Maybe add a sentence or two to explain the tunneling principle a bit more... (why I would do that? what is it?)